buttons-examples/Web-MWC/web-button-example.md

```diff
@@ -0,0 +1,457 @@
+
+Design Attribute | Theme value | CSS property
+---|---|---
+Ripple color | Primary color at 16% opacity | None (can override `--mdc-theme-primary`)
+Shape radius | 4px | None
+Outline color | Primary color (see note) | `--mdc-button-outline-color`
+Outline width | 1px | `--mdc-button-outline-width`
+Container width | Automatic | `width` 
+Horzontal padding | 16px - outline width | `--mdc-button-horizontal-padding`
+
+Note: The Material design spec calls for the outline to be the on-surface color at 12% opacity.
+Currently `mwc-button` defaults to the primary color for outlines.
+
+#### Icon attributes
+
+ Design Attribute | Theme value | HTML attribnute | CSS property
+---|---|---|---
+ Icon | N/A | `icon` | N/A
+Icon color | Primary color | N/A | None (can override `--mdc-theme-primary`)
+
+We recommend using [Material Theming](https://material.io/components/buttons/#theming) to apply your customizations across your application. For a full list of component properties, go to the [`mwc-button` API doc]().
+
+## Contained button
+
+[Contained buttons](https://material.io/components/buttons/#contained-button) are high-emphasis buttons, distinguished by their use of elevation and fill. They contain actions that are primary to your app.
+
+#### Contained button example
+
+API and source code:
+
+* `mwc-button`
+    * [API](https://github.com/material-components/material-components-web-components/blob/master/packages/button/README.md)
+    * [Demo](https://glitch.com/~many-witty-chihuahuas)
+    * [Source code](https://github.com/material-components/material-components-web-components/tree/master/packages/button/src)
+
+
+<img src="assets/web-contained-button.png" alt="Contained button example">
+
+```html
+<mwc-button raised label="Contained"></mwc-button>
+```
+
+#### Contained button with icon example
+
+<img src="assets/web-contained-icon-button.png" alt="Contained button with a bookmark icon">
+
+```html
+<mwc-button raised icon="bookmark" label="Contained Button plus Icon"></mwc-button>
+```
+</details>
+
+### Anatomy and key attributes
+
+A contained button has text, a container, and an optional icon.
+
+![Contained button anatomy diagram](assets/contained_button_anatomy.png)
+
+1. Text label
+1. Container
+1. Icon
+
+#### Text label attributes
+
+Design Attribute | Theme value | HTML attribute | CSS property
+---|---|---
+Text label | N/A | `label` | N/A
+Text color | On-primary color | N/A | None (can override `--mdc-theme-on-primary`)
+Typography | Default button text style | `--mdc-typography-font-family`, `--mdc-typography-button-*`
+
+For details on the available typography properties, see [Typography properties](../theming#typography-properties) in the Theming guide.
+
+#### Container attributes
+
+Design Attribute | Theme value | CSS Property
+---|---|---
+Container fill color| Primary color | None (can override `--mdc-theme-primary`)
+Ripple color | On-primary color at 32% opacity | None (can override `--mdc-theme-on-primary`)
```

Comment:
This one is actually hardcoded right now, blocked on https://github.com/material-components/material-components-web-components/issues/1072